### DISCUSSION TITLE
How to generate html files

### DISCUSSION THREAD

**Discussion by CppInterview** [vuepress/core#915] [Q&A] [answered]
As we know, using yarn docs:build to generate html files to dist directory.

But the html files did not contain the context we write in .md files but had a lot of .js files instead.

So my question is, how to generate html file had .md files content in it?

**Comment by CppInterview** [comment]
如果在chrome浏览器右键 -> 查看源代码，

看不到我们在.md文件中写入的内容，只有一些.js文件。

以我的认知，这样对seo不太友好。

有没有人知道原因？谢谢！

**Comment by Mister-Hope** [comment, marked as answer]
I saw this question and I decided to ignore it. But tif you insist, I will make a simple reply here.

You'd better learn some basic knowledge before asking this question..

Just providing an example for your question in others mind.

Someone is asking:

- Why the stock market use red for rising and green for falling, isn't that opposite with traffic light?
- Why don't they provide a count down before changing the price?

**Comment by CppInterview** [comment]
I found it's a false issue when I tried to reproduce it.

The html files contain .md files content in it.

I did not find it yet because the vuepress compress the app element in html file to one line.

![image](https://user-images.githubusercontent.com/106243057/171331758-b02c935e-40e8-444e-a658-fcf5c115aa49.png)

Thanks for your kind reminder, I am a newbee in github and not good at English expressions.

**Reply by Mister-Hope** [reply]
You'd better use words like markdown rendered content instead of ,md content, because the latter will be raw markdown content in most cases.

We use SSR to generate HTML files, so of course we output html structure for every page.
